modules/core/04-channel/keeper/migrations_test.go

```diff
@@ -0,0 +1,34 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/ibc-go/v8/modules/core/04-channel/keeper"
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+)
+
+// TestMigrateDefaultParams tests the migration for the channel params
+func (suite *KeeperTestSuite) TestMigrateDefaultParams() {
+	testCases := []struct {
+		name           string
+		expectedParams channeltypes.Params
+	}{
+		{
+			"success: default params",
```

Comment:
a test to cover the `if err := params.Validate()` case?

Reply:
just looking into this now, I don't think we can actually do this, since we are calling Validate directly on DefaultParams. I think we can actually remove the check instead. DefaultParams will always be valid